scripts/osfstorage/usage_audit.py

```diff
@@ -0,0 +1,98 @@
+"""File: usage_audit.py
+Find all users and projects where their total usage (current file + deleted files) is >= the set limit
+Projects or users can have their GUID whitelisted via `usage_audit whitelist [GUID ...]`
+User usage is defined as the total usage of all projects they have > READ access on
+Project usage is defined as the total usage of it and all its children
+total usage is defined as the sum of the size of all verions associated with X via OsfStorageFileNode and OsfStorageTrashedFileNode
+"""
+import os
+import sys
+import json
+import logging
+from collections import defaultdict
+
+from modularodm import Q
+from website import mails
+from website.app import init_app
+from website.project.model import Node
+from website.addons.osfstorage import model
+
+from scripts import utils as scripts_utils
+
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+GBs = 1024 ** 3.0
+
+USER_LIMIT = 5 * GBs
+PROJECT_LIMIT = 5 * GBs
+
+WHITE_LIST_PATH = os.path.join(os.path.dirname(__file__), 'usage_whitelist.json')
+
+
+try:
+    with open(WHITE_LIST_PATH, 'r') as fobj:
+        WHITE_LIST = set(json.load(fobj))  # Cast to set for constant time look ups
+    logger.info('Loaded whitelist.json from {}'.format(WHITE_LIST_PATH))
+except IOError:
+    WHITE_LIST = set()
+    logger.warning('No whitelist found')
+
+
+def add_to_white_list(gtg):
+    gtg = set(gtg).difference(WHITE_LIST)
+    logger.info('Adding {} to whitelist'.format(gtg))
+    with open(WHITE_LIST_PATH, 'w') as fobj:
+        json.dump(list(WHITE_LIST.union(gtg)), fobj)  # Sets are not JSON serializable
+    logger.info('Whitelist updated to {}'.format(WHITE_LIST))
+
+
+def get_usage(node):
+    addon = node.get_addon('osfstorage')
+    calculate_usage = lambda m: sum([v.size or 0 for fn in m.find(Q('node_settings', 'eq', addon)) for v in fn.versions])  # Sum all versions of all files of this node
```

Comment:
Again, avoid 1- or 2-letter variable names. For example `m` should be `model_class` or `model_cls`.